rust/repositories.bzl

```diff
@@ -656,6 +669,10 @@ rust_toolchain_repository = repository_rule(
         "sha256s": attr.string_dict(
             doc = "A dict associating tool subdirectories to sha256 hashes. See [rust_repositories](#rust_repositories) for more details.",
         ),
+        "urls": attr.string_list(
+            doc = "A list of mirror urls containing the tools from the Rust-lang static file server. These must contain the '{}' used to substitute the tool being fetched (using .format). It defaults to the official repository 'https://static.rust-lang.org/dist/{}.tar.gz'",
```

Comment:
Nit, I think users will see the default in the rendered docs

Reply:
should i remove the default part from the doc?

Reply:
I would, I know there are a bunch of similar docs for macro parameters, but I think it'd be better to keep them out of attribute `doc`s.

Reply:
done in fixup!